src/main.rs

```diff
@@ -92,7 +95,7 @@ fn get_json(extra_args: &[&str]) -> Result<String, ProgramError> {
     Ok(String::from_utf8(output.stdout)?)
 }
 
-fn handle_suggestions(suggestions: &[Suggestion]) -> Result<(), ProgramError> {
+fn handle_suggestions(suggestions: &[Suggestion], yolo: bool) -> Result<(), ProgramError> {
```

Comment:
Nope, this is not how we do things. There needs to be an `enum Assertiveness { BetterSafeThanSorry, Yolo }` instead of a boring `bool`! :trollface: